features/reward_features.py

```diff
@@ -291,6 +291,46 @@ def _start_tracking_out(self):
             super()._start_tracking_out()
             self.reward.off()
 
+
+class ScoreRewards(traits.HasTraits):
+    '''
+    Add a "score" to the task that keeps track of the number of successful trials and
```

Comment:
Does it keep track of # of successful trials or just the reach time of the most recent trial?

Reply:
Can you update the documentation to explain how longer reach time translates to a lower score?